src/main/java/frc/robot/commands/TwoGamePiecesThatEngage.java

```diff
@@ -85,9 +86,8 @@ public TwoGamePiecesThatEngage(
         new WaitUntilCommand(() -> lift.atPosition(LiftPosition.STARTING)),
         new FollowPathWithEvents(
             drive.followTrajectoryCommand(trajInit, true), trajInit.getMarkers(), eventMap),
-        /** TODO: Limelight code goes here 
-         * if limelight fails to find valid target/arpil tag, then turn on NO_TAG light
-        */
+        /** TODO: Limelight code goes here */
+        new InstantCommand(()->{if(tagLimelight.getValidTarget() == 0){lights.toggleCode(LightCode.NO_TAG);}}),
```

Comment:
Instead use `isValidTarget()` which returns a boolean (line 220 in TagLimelight)